packages/xo-server/src/vhd-merge.integ.spec.js

```diff
@@ -0,0 +1,203 @@
+/* eslint-env jest */
+
+import execa from 'execa'
+import fs from 'fs-extra'
+import path from 'path'
+import rimraf from 'rimraf'
+import LocalHandler from './remote-handlers/local.js'
+import { tmpDir, pFromCallback } from './utils'
+import vhdMerge, { chainVhd, Vhd } from './vhd-merge'
+
+const initialDir = process.cwd()
+
+jest.setTimeout(10000)
+
+beforeEach(async () => {
+  const dir = await tmpDir()
+  process.chdir(dir)
+})
+
+afterEach(async () => {
+  const tmpDir = process.cwd()
+  process.chdir(initialDir)
+  await pFromCallback(cb => rimraf(tmpDir, cb))
+})
+
+async function createRandomFile (name, sizeMo) {
+  await execa('bash', [
+    '-c',
+    `< /dev/urandom tr -dc "\\t\\n [:alnum:]" | head -c ${sizeMo}M >${name}`,
+  ])
+}
+
+async function checkFile (vhdName) {
+  await execa('vhd-util', ['check', '-p', '-b', '-t', '-n', vhdName])
```

Comment:
Is this command part of the qemu-utils package?

Reply:
no, it's blktap-utils (it took a bit of time to find it)